Frontend/config.js

```diff
@@ -0,0 +1,5 @@
+const config = {
+    apiBaseUrl: 'http://localhost:3000'
+  };
+  
+  export default config;
```

Comment:
nit: I think this file's format is a little off, need to format again

Reply:
I just fixed it. Was that the formatting change you were referring to?